config/settings/roles.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.admin import UserAdmin
```

Comment:
lets remove unused imports
I dont think we need this here

Reply:
Noted